u-root.go

```diff
@@ -332,3 +352,37 @@ func Main(l ulog.Logger, buildOpts *gbbgolang.BuildOpts) error {
 	}
 	return uroot.CreateInitramfs(l, opts)
 }
+
+func validateArg(arg string) bool {
+	if !checkPrefix(arg) {
+		if strings.Contains(arg, "*") {
```

Comment:
Glob can contain other matching characters, like ? or [a-z]. I think you can simplify this to just:

```go
if !checkPrefix(arg) {
  paths, err := filepath.Glob(arg)
  if err != nil {
    return false
  }
  for _, path := range paths {
    if !checkPrefix(path) {
      return false
    }
  }
}
return true
```

no recursion necessary, and no string.Contains.